src/roiextractors/extractors/memmapextractors/memmapextractors.py

```diff
@@ -64,43 +62,86 @@ def get_num_channels(self):
         return self._num_channels
 
     @staticmethod
-    def write_imaging(imaging_extractor: ImagingExtractor, save_path: PathType = None, verbose: bool = False):
+    def write_imaging(
+        imaging_extractor: ImagingExtractor,
+        save_path: PathType = None,
+        verbose: bool = False,
+        force_chunk: bool = False,
```

Comment:
For consistency, it might just be nice to follow either the MovieInterface or RecordingInterfaces as much as possible.

The RecordingInterfaces use `iterator_opts` which is a dictionary passed directly as kwargs, including the `buffer_gb`, which can be set to larger than the file size, which has the equivalent effect of `force_chunk=False` here.

The MovieInterface is more similar, a boolean option called `chunk_data`: https://github.com/catalystneuro/nwb-conversion-tools/blob/main/src/nwb_conversion_tools/datainterfaces/behavior/movie/moviedatainterface.py#L110, though it is recommended to be `True` by default (in general, just a good practice I've found)

Reply:
Good idea. I will follow `MovieInterface`.

Reply:
Done.

Reply:
Also, it should really probably be called 'buffer_data', even in the MovieInterface. 'Chunk' gets conflated when talking about binary vs. HDF5, so 'buffer' is more accurate to what we're doing.

Reply:
OK, chunking is confusing to me I have to say.

The thing that we are doing is something like:
"Loading the data in a small chunks by means (by means of a buffer?) so we don't overload memory" 

I wonder if there is a better way of denoting that.

Reply:
Done.